### PR TITLE
[Laravel 5.8] Add MinutesToSecondsInCacheRector + misc

### PR DESCRIPTION
**Pull request by TomasVotruba**
Ref https://github.com/laravel/framework/pull/27276/files

### CODE REVIEW

**Comment by TomasVotruba** [comment]
cc @barryvdh I think this is what you needed :)

Could you give me code-review and complete more cases if needed?

**Comment by barryvdh** [comment]
My preference would go to actually using the Datetime instance, but maybe that can be configurable? https://twitter.com/laravelshift/status/1097514254016696323

File: packages/Laravel/src/Rector/StaticCall/MinutesToSecondsInCacheRector.php [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+        return new RectorDefinition(
+            'Change minutes argument to seconds in Illuminate\Contracts\Cache\Store and Illuminate\Support\Facades',
+            [
+                new CodeSample(
+                    <<<'CODE_SAMPLE'
+class SomeClass
+{
+    public function run()
+    {
+        Illuminate\Support\Facades::put('key', 'value', 60);
```
Comment (barryvdh) [inline packages/Laravel/src/Rector/StaticCall/MinutesToSecondsInCacheRector.php]: Missing 'Cache' here

File: packages/Laravel/src/Rector/StaticCall/MinutesToSecondsInCacheRector.php [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+    }
+}
+CODE_SAMPLE
+                    ,
+                    <<<'CODE_SAMPLE'
+class SomeClass
+{
+    public function run()
+    {
+        Illuminate\Support\Facades::put('key', 'value', 3600);
```
Comment (barryvdh) [inline packages/Laravel/src/Rector/StaticCall/MinutesToSecondsInCacheRector.php]: You actually modifying it to 60*60, right?

File: packages/Laravel/src/Rector/StaticCall/MinutesToSecondsInCacheRector.php [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+
+        return $node;
+    }
+
+    private function processArgumentPosition(StaticCall $staticCall, int $argumentPosition): StaticCall
+    {
+        if ($staticCall->args[$argumentPosition]->value instanceof LNumber) {
+            // number
+            $value = $this->getValue($staticCall->args[$argumentPosition]->value);
+            if ($value === 60) {
```
Comment (barryvdh) [inline packages/Laravel/src/Rector/StaticCall/MinutesToSecondsInCacheRector.php]: Why only 60?

**Comment by barryvdh** [comment]
Although when you use * 60 it's pretty clear.

File: packages/Laravel/src/Rector/StaticCall/MinutesToSecondsInCacheRector.php (hunk 1 above)
Comment (TomasVotruba) [reply packages/Laravel/src/Rector/StaticCall/MinutesToSecondsInCacheRector.php]: Thanks :+1:

File: packages/Laravel/src/Rector/StaticCall/MinutesToSecondsInCacheRector.php (hunk 3 above)
Comment (TomasVotruba) [reply packages/Laravel/src/Rector/StaticCall/MinutesToSecondsInCacheRector.php]: Just first idea, no reason really :) fixed

File: packages/Laravel/src/Rector/StaticCall/MinutesToSecondsInCacheRector.php (hunk 2 above)
Comment (TomasVotruba) [reply packages/Laravel/src/Rector/StaticCall/MinutesToSecondsInCacheRector.php]: Yes, later I conclude it's more readable. 

I forgot to fix it here, thanks :+1:

**Comment by TomasVotruba** [comment]
> My preference would go to actually using the Datetime instance, but maybe that can be configurable? twitter.com/laravelshift/status/1097514254016696323

That should be handled by another rule, so it's consisten through whole code, like `laravel-coding-style`.

**Comment by barryvdh** [comment]
Yeah perhaps you're right. The main issue was that it wasn't totally clear if it's converted or not, but with * 60 that isn't an issue. So if this is in the upgrade  set it's fine, and indeed a code style set would be better for 5.8+

**Comment by TomasVotruba** [comment]
Ready to merge for me

**Comment by barryvdh** [comment]
Looks good to me too :) Thanks!

**Pull request merged**